docs/CONTRIBUTING

```diff
@@ -0,0 +1,8 @@
+# Contributions to Satisfactory Factories
+
+Hello and thank you for your interest in contributing to Satisfactory Factories! We are excited to have you here and appreciate your help in making this project better.
+
+## How to Contribute
+Please read the documents contained within the `/docs` folder to get started.
+
+More information on how to spin up your project locally can be found in the [README](../README.md).
```

Comment:
I think it would be useful to also post something here about:

1. Contributions are welcome. Start a PR! 
2. Don't start a massive feature without creating an issue and getting confirmation this fits into the strategy of the project.  

Also, something similar to this is important too (which really circles back to - if you aren't sure, create an issue or ask on the discord). (I did not write this, I `lifted` if from an open source blog I follow:

>Keep in mind that there is no democracy in software projects. Despite the product/library being built by many people around the world, there will be a small group that will guarantee the integrity of the repository and will decide what should be merged or not. Accept this as an axiom and don’t let it hurt your ego.

Reply:
@samsmithnz updated, let me know what you think!